paper-checkbox.html

```diff
@@ -40,9 +40,12 @@
 `--paper-checkbox-checked-ink-color` | Selected/focus ripple color when the input is checked | `--primary-color`
 `--paper-checkbox-checkmark-color` | Checkmark color | `white`
 `--paper-checkbox-label-color` | Label color | `--primary-text-color`
+`--paper-checkbox-label-checked-color` | Label color when the input is checked | (none)
```

Comment:
Oh, "(none)" was a poor word choice here on my part given that it's also CSS value. There's intentionally no default so that when this value is not set it inherits from `--paper-checkbox-label-color`. Maybe I should just make it `--paper-checkbox-label-color` explicitly - that should work the same way.